src/Symfony/Installer/DemoCommand.php

```diff
@@ -155,6 +156,33 @@ private function displayInstallationResult()
     }
 
     /**
+     * Updates the composer.json file to provide better values for some of the
+     * default configuration values.
+     *
+     * @return $this
+     */
+    protected function updateComposerJson()
```

Comment:
Good question! I've refactored everything to move this method to the abstract parent class.